build/configure.js

```diff
@@ -145,19 +141,77 @@ function buildRules(grunt, options, commons, callback) {
 
 				return c.options === undefined ? id : c;
 			});
-
 		}
 
+		function parseImpactForRule(rule) {
+			
+
+			function capitalize(s) {
+				return s.charAt(0).toUpperCase() + s.slice(1);
+			}
+
+			function getUniqueArr(arr) {
+				return arr.filter(function(value, index, self) { 
+					return self.indexOf(value) === index;
+				})
+			}
+
+			function getImpactScores(checkCollection) {
+				return checkCollection.reduce(function (out, check) {
+					var id = typeof check === 'string' ? check : check.id;
+					var definition = clone(findCheck(checks, id));
+					if (!definition) {
+						grunt.log.error('check ' + id + ' not found');
+					}
+					if (definition && definition.metadata && definition.metadata.impact) {
+						var impactScore = axeImpact.indexOf(definition.metadata.impact);
+						out.push(impactScore);
+					}
+					return out;
+				}, []);
+			}
+
+			function getScore(checkCollection, onlyHighestScore) {
+				var scores = getImpactScores(checkCollection);
+				if(scores && scores.length) {
+					return onlyHighestScore
+						? [Math.max.apply(null, scores)]
+						: getUniqueArr(scores);
+				} else{
+					return [];
+				}
+			}
+
+			var cumulativeScores = getUniqueArr(
+				getScore(rule.any, true) 							// Any: get highest impact
+					.concat(getScore(rule.all, false)) 	// All: get all unique
+					.concat(getScore(rule.none, false)) // None: get all unique
+			).sort().reverse(); //order highest to lowest
```

Comment:
I would think lowest to highest. Not a big deal though.

Reply:
Sorted - lowest to highest...